client/driver/OracleConf.py

```diff
@@ -0,0 +1,45 @@
+#
+# OtterTune - OracleConf.py
+#
+# Copyright (c) 2017-18, Carnegie Mellon University Database Group
+#
+'''
+Created on Feb 20, 2019
```

Comment:
Delete this comment (I realize there's a similar comment in PostgresConf.py but it's unnecessary)